xml/templates/civicrm_state_province.tpl

```diff
@@ -4163,7 +4163,21 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 -- Add in missing Indian State provinces
 
 (NULL, 1101, "DH", "Dādra and Nagar Haveli and Damān and Diu"),
-(NULL, 1101, "LA", "Ladākh")
+(NULL, 1101, "LA", "Ladākh"),
 
  -- end of statement:
+
+-- Add missing provinces for Luxembourg
+(NULL, 1020, 'CA', 'Capellen'),
```

Comment:
@seamuslee001 @jmacarthur98 This is the wrong country id - it's Belgium which also has a state called Luxembourg. Should be 1126.

@seamuslee001 You could fix this in your PR.

Reply:
thanks @demeritcowboy will fix and noticed also this was missing the fixing of the original state provinces's   abbreviations so will fix that as well